EventHubs/tests/baseeventhubtest.py

```diff
@@ -1,33 +1,35 @@
 import os
 import sys
-sys.path.insert(0, '../../test_utils')
-from basetest import BaseTest
 import json
+import time
 from time import sleep
 from requests import Session
+from datetime import datetime
+from azure.servicebus import ServiceBusService
+from azure.identity import DefaultAzureCredential
 from azure.mgmt.resource import ResourceManagementClient
 from azure.mgmt.eventhub import EventHubManagementClient
-from azure.mgmt.storage import StorageManagementClient
-from azure.cosmosdb.table.tableservice import TableService
-from azure.servicebus import ServiceBusService
 
+sys.path.insert(0, '../../test_utils')
+from basetest import BaseTest
 
 class BaseEventHubTest(BaseTest):
 
     def setUp(self):
-        self.create_credentials()
-        self.resource_client = ResourceManagementClient(self.credentials,
-                                                        self.subscription_id)
+        self.azure_credential = DefaultAzureCredential()
```

Comment:
self.azure_credential = DefaultAzureCredential()
self.subscription_id = os.getenv("AZURE_SUBSCRIPTION_ID")
put this in base test otherwise other test will fail

Reply:
updated